fillin/src/main/java/com/teamfillin/fillin/domain/user/Status.java

```diff
@@ -0,0 +1,5 @@
+package com.teamfillin.fillin.domain.user;
+
+public enum Status {
```

Comment:
UserStatus 로 네이밍이 명확하면 좋을것같습니다~
```suggestion
public enum UserStatus {
```

Status 라는 네이밍을 사용한다면, public 을 제거해 user package 안에서만 접근가능하게 해야할것같아요~

```suggestion
enum Status {
```

Reply:
UserStatus 로 가져가겠습니다!!